Mogien/MVC_Git
# Static page

# General Description
Asana issue : https://app.asana.com/0/1111428436201563/1203002942462051/f

## Prerequisites / How to Test
1. Create new repo in github and clone it your local machine`
$ git clone github repository link
2. Create new C# MVC project in visual studio
3. Save it into your local repository
Step 1. To check file status
$ git status
Step 2. Then add file to commit
$ git add .
5. Commit and push the changes to github repository
Step 3. Then commit file
$ git commit -m "Your message"
Step 4. Then push file to repository
$ git push origin master or main

Implement static page
6. Before you modify the home page, please create a new branch
- Step 5. To view branch
$ git branch
- Step 6. To create new branch
$ git branch branch_name
- Step 7. To checkout branch
$ git checkout branch_name
8. Modify the home page to view list of Orders
  - Order No
  - Order Date
  - Customer Name
  - Total Amount
9. When your done with item #9, go ahead and commit the changes
- Step 8. Follow step 1 to 4
Visual Studio Side: When the user click the Order No, it will open the Order Detail page that will show the information in a form view. Details page will have Back button to go back to Orders page.
10. If done, follow step 1 to 4.

PS: do not include /bin and /obj folders, it should not be saved in the repository
To ignore bin/obj
Go to GIT BASH
Then type "touch .gitignore"
Check folder if was file created.
Inside the gitignore, type the path of the folder/file that do you want to ignore.
Before to add the ignore to commit.
Clear the cache first, type "git rm -r --cached ."
Then follow step 1 to 4.
Then check your repository.
